src/openklant/components/klantinteracties/api/viewsets/klantcontacten.py

```diff
@@ -331,6 +332,20 @@ def perform_create(self, serializer):
             token_identifier=token_auth.identifier,
             token_application=token_auth.application,
         )
+        object_type = instance.onderwerpobjectidentificator.get("code_objecttype")
+        if object_type == "zaak":
```

Comment:
Could you add the setting `ENABLE_CLOUD_EVENTS` (same as https://github.com/open-zaak/open-zaak/blob/2be1bc48a93107205a9148f2899bc33f810fd7fa/src/openzaak/conf/includes/base.py#L531-L537) and make sure the event is only sent if this setting is True?

You can remove the `add_to_docs=False` part